recipes/segregation/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Can you remove these comments from the example

Reply:
Of course, just solved this in the last commit! Please let me know if anything else is needed!